e2e/e2etests/helpers.go

```diff
@@ -90,6 +65,44 @@ func withdrawBTCZRC20(r *runner.E2ERunner, to btcutil.Address, amount *big.Int)
 	return rawTx
 }
 
+// BTCWithdraw is a helper function to call 'withdraw' on BTCZRC20 contract with optional 'approve'
+func BTCWithdraw(r *runner.E2ERunner, to btcutil.Address, amount *big.Int, approve bool) *ethtypes.Receipt {
```

Comment:
Let's make this runner's method in `runner/bitcoin.go` (similar to other chains)
```suggestion
func (r *runner.E2ERunner) BTCWithdraw(to btcutil.Address, amount *big.Int, approve bool) *ethtypes.Receipt {
```

Reply:
https://github.com/zeta-chain/node/pull/3417/commits/142e0a459eb2f993a014328f95c5cc43a7f385ab